src/couch_mrview/test/couch_mrview_ddoc_updated_tests.erl

```diff
@@ -47,12 +46,17 @@ setup() ->
     {ok, _} =
         couch_mrview:query_view(Db2, <<"_design/bar">>, <<"baz">>, [], CB, 0),
 
+    meck:expect(couch_index_updater, update, fun(Idx, Mod, IdxSt) ->
```

Comment:
I *think* you need to add a line meck:new(couch_index_updater, [passthrough]) so that this expectation can use the meck:passthrough/1 function.